### ISSUE TITLE
Accessibility Flag - Element IDs are not unique

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/SearchWorks#3768]
SiteImprove throws the following error: Element IDs are not unique. Two or more elements on this page have the same id value. These should be unique.

On the following page(s): https://searchworks.stanford.edu/articles (specifically ferencing the recaptcha section of the site) 

Why is this a problem?
https://www.w3.org/TR/WCAG20-TECHS/F77.html#F77-disclaimer

**Comment by jcoyne** [comment]
@dbranchini can you show me where you see that in Siteimprove? I'm having trouble finding it there.

**Comment by dbranchini** [comment]
It's picking up (and I was able to manually find too) two #feedback-form-recaptcha elements.

**Comment by jcoyne** [comment]
@dbranchini can you show me how you are finding that in siteimprove?

**Comment by jcoyne** [comment]
<img width="400" alt="Screenshot 2024-01-23 at 11 21 27 AM" src="https://github.com/sul-dlss/SearchWorks/assets/92044/9e905516-f317-4798-b406-ef45f83ea75e">
It is still not complete.
